src/crimson/os/seastore/cache.h

```diff
@@ -1741,6 +1751,8 @@ class Cache {
           extent->on_clean_read();
         }
 
+        lru.maybe_increase_cached_size(
+          *extent, extent->get_loaded_length() - old_length);
```

Comment:
Seems wrong to adjust LRU size and change the actual loaded_length(in `extent->load_range()`) separately.

I think they need to be atomic because otherwise if the extent is removed in the middle (`lru.remove_from_lru(extent)`), the LRU size will become wrong because `extent->get_loaded_length()` might have been changed implicitly.

How about following the original design to 1.create the buffer, 2.increase extent size and 3.increase lru size atomically at the begining of `Cache::read_extent()` ?